infrastructure/docker-compose-postgres.yml

```diff
@@ -2,7 +2,7 @@
 # provides a Postgresql database
 # DB: login = postgres:postgres; hostname = postgres; port = 5432; volume = psqldata_iris
 ########################################################
-version: '3'
+version: '2.3'
```

Comment:
With version 3 I get the following error (docker-compose 1.25.0):
```
ERROR: The Compose file './docker-compose-postgres.yml' is invalid because:
services.postgres.healthcheck value Additional properties are not allowed ('start_period' was unexpected)
```
The start_period parameter seems to be removed in version 3. With 2.3 it works.

Reply:
You use a docker-compose version from nov. 2019!

Reply:
`start_period` is also part of the file version 3 since 3.4: https://docs.docker.com/compose/compose-file/compose-versioning/#version-34.